docs/README.md

```diff
@@ -29,12 +29,16 @@ Feathr automatically computes your feature values and joins them to your trainin
 - **Native cloud integration** with simplified and scalable architecture, which is illustrated in the next section.
 - **Feature sharing and reuse made easy:** Feathr has built-in feature registry so that features can be easily shared across different teams and boost team productivity.
 
-## Running Feathr on Azure with few simple steps
+## ☁️ Running Feathr on Cloud with a few simple steps
+
+Feathr has native integrations with Databricks and Azure Synapse:
 
 Follow the [Feathr ARM deployment guide ](https://linkedin.github.io/feathr/how-to-guides/azure-deployment-arm.html) to run Feathr on Azure. This allows you to quickly get started with automated deployment using Azure Resource Manager template.
 
 If you want to set up everything manually, you can checkout the [Feathr CLI deployment guide](https://linkedin.github.io/feathr/how-to-guides/azure-deployment-cli.html) to run Feathr on Azure. This allows you to understand what is going on and set up one resource at a time.
 
+- Please read the [Quick Start Guide for Feathr on Databricks](./quickstart_databricks.md) to run Feathr with Databricks.
```

Comment:
I remember it's recommended to use html links in docs, shall we update it in this PR?

Reply:
I'm actually not sure about that given that the relative links works better here (less maintenance cost) and after compiling, the relative links will turn into absolute links.